sidebars.js

```diff
@@ -161,51 +134,122 @@ module.exports = {
+          "operate-a-stake-pool/node-operations/cardano-components",
+          "operate-a-stake-pool/node-operations/installing-cardano-node",
+          "operate-a-stake-pool/node-operations/running-cardano",
+          "operate-a-stake-pool/node-operations/dynamic-block-forging", 
+          "operate-a-stake-pool/node-operations/rts-options-node",
+          "operate-a-stake-pool/node-operations/topology",
+          {
+            type: "category",
+            label: "New Tracing System",
+            items: [
+              {
+                type: "autogenerated",
+                dirName: "operate-a-stake-pool/node-operations/new-tracing-system",
+              },
+            ],
+          },
         ],
       },
       {
         type: "category",
-        label: "Cardano Relay Configuration",
+        label: "Cardano CLI",
         items: [
-          "operate-a-stake-pool/relay-node-configuration",
-          "operate-a-stake-pool/monitoring-gLiveView",
-          "operate-a-stake-pool/grafana-dashboard-tutorial",
+          {
+            type: "category",
+            label: "Basic Operations",
+            link: {
+              type: "doc",
+              id: "operate-a-stake-pool/cli-operations/basic-operations/get-started"
+            },
+            items: [
+              {
+                type: "autogenerated",
+                dirName: "operate-a-stake-pool/cli-operations/basic-operations",
+              },
+            ],
+          },
+          "operate-a-stake-pool/cli-operations/simple-scripts/simple-scripts",
+          "operate-a-stake-pool/cli-operations/native-assets/native-assets", 
+          "operate-a-stake-pool/cli-operations/plutus-scripts/plutus-scripts",
+          {
+            type: "category",
+            label: "CLI - Governance",
+            items: [
+              {
+                type: "autogenerated",
+                dirName: "operate-a-stake-pool/cli-operations/governance",
+              },
+            ],
+          },
         ],
       },
       {
         type: "category",
-        label: "Cardano Block Producer Configuration",
+        label: "Cardano Wallet",
         items: [
-          "operate-a-stake-pool/generating-wallet-keys",
-          "operate-a-stake-pool/block-producer-keys",
-          "operate-a-stake-pool/register-stake-address",
-          "operate-a-stake-pool/register-stake-pool",
+          {
+            type: "autogenerated",
+            dirName: "operate-a-stake-pool/cardano-wallet",
+          },
         ],
       },
       {
         type: "category",
-        label: "Deployment Scenarios",
+        label: "Key Management",
         items: [
-          "operate-a-stake-pool/hardening-server",
-          "operate-a-stake-pool/improve-grafana-security",
-          "operate-a-stake-pool/audit-your-node",
+          {
+            type: "autogenerated",
+            dirName: "operate-a-stake-pool/security",
+          },
```

Comment:
... i.e. if currently left where they are, the introduction to the page would only appear applicable to SPOs who are ***also* interested** in minting assets & NFTs ... hence the original agreement when this material was posted to put it "higher up" in the developer introduction & have the SPO documentation link to it as necessary.

Reply:
Resolving as per https://github.com/cardano-foundation/developer-portal/pull/1609#pullrequestreview-3058421641 so this can be merged.